keras/src/layers/layer.py

```diff
@@ -1697,6 +1697,58 @@ def rematerialized_activation_call_wrapper(*args, **kwargs):
                 return rematerialized_activation_call_wrapper
         return layer_call
 
+    def register_call_context_args(self, *names):
```

Comment:
This API is basically single-user, so please keep it underscore-private

Reply:
Done